fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/MockBrowsingDataHelper.kt

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.helpers
+
+import android.content.Context
+import androidx.test.platform.app.InstrumentationRegistry
+import kotlinx.coroutines.runBlocking
+import mozilla.appservices.places.BookmarkRoot
+import mozilla.components.browser.icons.IconRequest
+import mozilla.components.browser.icons.generator.DefaultIconGenerator
+import mozilla.components.browser.state.search.SearchEngine
+import mozilla.components.browser.storage.sync.PlacesBookmarksStorage
+import mozilla.components.browser.storage.sync.PlacesHistoryStorage
+import mozilla.components.concept.storage.PageVisit
+import mozilla.components.concept.storage.VisitType
+import mozilla.components.feature.search.ext.createSearchEngine
+import okhttp3.mockwebserver.MockWebServer
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.helpers.TestHelper.appContext
+import org.mozilla.fenix.search.SearchEngineSource.None.searchEngine
+
+object MockBrowsingDataHelper {
```

Comment:
Wondering if there's a better context to rename this class as it uses storage, state, and search, maybe just 
`MockBrowserDataHelper`